core/homophones/homophones_open.talon

```diff
@@ -1,10 +1,10 @@
 tag: user.homophones_open
 -
 choose <number_small>:
-    result = user.homophones_select(number_small)
+    result = user.choose(number_small)
     insert(result)
     user.homophones_hide()
```

Comment:
this whole thing needs to be the `choose` action, and then you would remove this whole command entirely. The way you have it, you've created a nondeterministic conflict between the `choose <number_small>` command here and the global one

Reply:
I think I fixed that now. I did discover that the type signatures disagreed so I had to add a return type to the default choose implementation, which I wasn't sure about.

Reply:
Nevermind. It may have been what you originally meant, but at least I figured out a way that seems more simple.

Reply:
If I'm still not getting it, please feel free to push to this branch.